src/wallet/wallet2.cpp

```diff
@@ -5037,6 +5044,32 @@ uint64_t wallet2::adjust_mixin(uint64_t mixin)
   return mixin;
 }
 //----------------------------------------------------------------------------------------------------
+uint32_t wallet2::adjust_priority(uint32_t priority)
+{
+  if (priority == 0 && get_default_priority() != 1 && auto_low_priority())
```

Comment:
Same here, don't use magic numbers.

Reply:
see above

Reply:
That case is a bit of a special one, since the semantics depend on which hardfork we're on, and as stoffu says it indexes in a fork specific table. Maybe defines would still be better (not sure), but since the rest of the code uses numbers right now, numbers are fine for this patch.